src/components/TodoItem.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable */
+import React, { useState, useEffect } from 'react';
+import { FaTrash } from 'react-icons/fa';
+import styles from './TodoItem.module.css';
+
+const TodoItem = (props) => {
+  const [editing, setEditing] = useState(false);
+
+  useEffect(() => () => {
+    console.log('Cleaning up...');
```

Comment:
Kindly ensure that there are no `console.log()` statements in your final production version of code. They are unnecessary to the end user and would produce undesirable behavior depending on the information being logged. 
Kindly only use these statements to debug your code.

Reply:
@Ellon-M I have removed console.log at this [commit](https://github.com/Lucash2022/react-todo-app/commit/0c56c664a14eacd737be0c59d206e59f4c219b57)